django-stubs/test/testcases.pyi

```diff
@@ -109,14 +109,15 @@ class SimpleTestCase(unittest.TestCase):
         msg_prefix: str = ...,
     ) -> None: ...
     @overload
-    def assertFormError(
+    def assertFormError(  # old signature, deprecated in Django 4.1
         self,
         response: HttpResponseBase,
         form: str,
         field: str | None,
         errors: list[str] | str,
         msg_prefix: str = ...,
     ) -> None: ...
+    # assertFormsetError (lowercase “set”) deprecated in Django 4.2
```

Comment:
Seems weird to have curly quotes in source code IMO. Let's avoid Unicode where it's not necessary.

```suggestion
    # assertFormsetError (lowercase "set") deprecated in Django 4.2
```

Reply:
Sure, force of habit from writing!